Content.Server/Afk/AFKSystem.cs

```diff
@@ -85,6 +85,8 @@ public override void Update(float frameTime)
 
         foreach (var session in Filter.GetAllPlayers())
         {
+            if (session.Status != SessionStatus.InGame) return;
```

Comment:
should be a continue.

Also when are people marked as ingame.

Reply:
Fixed continue